v1/brokers/sqs/sqs.go

```diff
@@ -59,6 +60,9 @@ func New(cnf *config.Config) iface.Broker {
 
 // StartConsuming enters a loop and waits for incoming messages
 func (b *Broker) StartConsuming(consumerTag string, concurrency int, taskProcessor iface.TaskProcessor) (bool, error) {
+	if concurrency < 1 {
+		concurrency = runtime.NumCPU() * 2
```

Comment:
Better to use `runtime.GOMAXPROCS(0)` so project with custom  GOMAXPROCS will use the number of allowed CPU

Reply:
Done. Thanks for the comment.